### ISSUE TITLE
convert to tflite model

### ISSUE THREAD

**Issue by cswwp** [tensorflow/models#5778]
I just want convert  mobilenet v2 without fc to tflite model, anyone help me ?

**Comment by SiddhantKapil** [comment]
What help do you want? Please be more specific?

**Comment by jillelajitta** [comment]
Hi @SiddhantKapil 
I parsed out 90 classes from OPEN IMAGE DATASET for object detection. I roughly have 600k train images and 200k val images.
I have trained this dataset on coco pretrained inception v2 model for 400k steps,at 200k steps it is only detecting smaller objects, after 300k steps it is only detecting larger objects. Could you please help me where I am going wrong.
How many steps do you suggest me to train? What is the best learning rate to start with.

**Comment by SiddhantKapil** [comment]
Hi @jillelajitta, I think that you have asked this question on the wrong thread. Please open another issue for it.

**Comment by cswwp** [comment]
I have solve this issue with follow codes
`frozen_graphdef = tf.graph_util.convert_variables_to_constants(sess, sess.graph_def,
                                                                           ['MobilenetV2/Predictions/Reshape_1']) 
            # pb model saving
            with tf.gfile.FastGFile(model_output + base_save_model + '_classify.pb', mode='wb') as f:
                f.write(frozen_graphdef.SerializeToString())`

**Comment by SiddhantKapil** [comment]
Just wondering why simply using TOCO to convert pb graph to tflite doesn't work?

**Comment by ymodak** [comment]
Closing since it is resolved. Feel free to reopen if the issue still persists. Thanks!
